pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -263,6 +296,6 @@ in stdenv.mkDerivation {
     license = licenses.lgpl21Plus;
     platforms = platforms.linux;
     priority = 10;
-    maintainers = with maintainers; [ andir eelco flokli mic92 ];
```

Comment:
do we want to bump `passthru.interfaceVersion`?

(grr github doesnt allow me to comment on unaffected parts of the diff. so stupid)

Reply:
Is there any reason to do that? At least since b20a0e49c8c0d5c84b917bfbbe3f2ead31886a67 `systemd` is able to properly re-exec itself (I just tested this locally and didn't experience any issues).

Reply:
Bumping this basically forces users to reboot, instead of re-exec'ing. As we seem to be able to re-exec, there's no point in bumping it.